app/controllers/users_controller.rb

```diff
@@ -23,10 +23,15 @@ def update
     end
   end
   
+  def videos
+    plays_tv = PlaysTv.new
+    @response = PlaysTv.get_videos('burnNturn3')
```

Comment:
Do not want to have this hardcoded for you everytime! :)

Reply:
opps, thought I changed that already. lol